powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/PowerAuthClient.java

```diff
@@ -1154,16 +1155,19 @@ VaultUnlockResponse unlockVault(String activationId, String applicationKey, Stri
     /**
      * Create a new callback URL with given parameters.
      *
-     * @param applicationId  Application ID.
-     * @param name           Callback URL display name.
-     * @param type           Callback type.
-     * @param callbackUrl    Callback URL value.
-     * @param attributes     Attributes to send in the callback data.
-     * @param authentication Callback request authentication.
+     * @param applicationId   Application ID.
+     * @param name            Callback URL display name.
+     * @param type            Callback type.
+     * @param callbackUrl     Callback URL value.
+     * @param attributes      Attributes to send in the callback data.
+     * @param authentication  Callback request authentication.
+     * @param retentionPeriod Duration after which a completed callback event is automatically removed.
+     * @param initialBackoff  Initial delay before retry attempt following a callback event failure.
+     * @param maxAttempts     Maximum number of attempts to send a callback event.
      * @return Information about new callback URL object.
      * @throws PowerAuthClientException In case REST API call fails.
      */
-    CreateCallbackUrlResponse createCallbackUrl(String applicationId, String name, CallbackUrlType type, String callbackUrl, List<String> attributes, HttpAuthenticationPrivate authentication) throws PowerAuthClientException;
+    CreateCallbackUrlResponse createCallbackUrl(String applicationId, String name, CallbackUrlType type, String callbackUrl, List<String> attributes, HttpAuthenticationPrivate authentication, Duration retentionPeriod, Duration initialBackoff, Integer maxAttempts) throws PowerAuthClientException;
```

Comment:
> Carthago delenda est

In long term, I prefer using methods with request object only and mark these as _deprecated_.